PWGJE/FlavourJetTasks/AliAnalysisTaskHFJetIPQA.cxx

```diff
@@ -1026,6 +1026,13 @@ Bool_t AliAnalysisTaskHFJetIPQA::Run(){
                                 sImpParXYZ.clear();
                                 sImpParXYZSig.clear();
                             }//end jetloop
+                            static bool isFirstEvent=kTRUE;
```

Comment:
Please avoid static variables. The EMCAL framework offers a function UserExecOnce which UserTaskd can overwrite in order to call Initialisation which need event information (i.e. run number).